workflow-service/src/main/java/com/redhat/parodos/workflow/execution/dto/WorkFlowRequestDTO.java

```diff
@@ -78,6 +80,15 @@ public static class ArgumentRequestDTO {
 
 			String value;
 
+			public void setValue(Object value) {
```

Comment:
Is this used at all?

Reply:
yes,  now the multi-select will return a list of string,  I need to deserialize it to string here

Reply:
I changed to string.valueof to avoid some parsing issue with quotes